django/contrib/admin/utils.py

```diff
@@ -319,6 +320,25 @@ def lookup_field(name, obj, model_admin=None):
     return f, attr, value
 
 
+def get_query_string(params, new_params=None, remove=None):
+    if new_params is None:
+        new_params = {}
+    if remove is None:
+        remove = []
+    p = params.copy()
```

Comment:
Still not found of the one-letter variable names ;-)
`p`, `k`, `r`...

Even though I know you are just moving pre-existing code; if this piece of code is being touched, it's probably a good time to also make it more readable